### ISSUE TITLE
Splitting on background not working when mapped on controller

### ISSUE THREAD

**Issue by DocRider** [LiveSplit/LiveSplit#897]
I usually set split buttons on my Xbox One DPAD, and it worked well.  But since I've formatted my computer, the controller is recognized when I'm on livesplit, but when I click on another window (my game, for example), the controller isn't recognized anymore on livesplit. It works perfectly with the keyboard, but since it's more handy for me to have the split buttons on my controller ...

Don't know if anyone had the same issue. And if you need more informations, feel free to ask :)

**Comment by wooferzfg** [comment]
sounds like #517 ?

**Comment by DocRider** [comment]
Yes, it looks like the same problem. So i checked if I had the problematic windows update installed (kb3140743) to uninstall it, but it doesn't appear in my Update list, as shown on the picture

![2017-10-12_185939](https://user-images.githubusercontent.com/32715644/31508866-baa37596-af7f-11e7-9209-a1047a5398f9.png)

So I don't know if it is included on the last update I have (because kb3140743 seems to be dated on March) and issues have been corrected or if it is still installed but for an unknown reason, invisible.

**Comment by CryZe** [comment]
I would guess every new install of Windows has this update. So we probably have to figure out some way to get it working with the update.

**Comment by just-ero** [comment]
Does this issue persist in the most recent stable version of LiveSplit?

**Comment by wooferzfg** [comment]
Yes, we haven't fixed it
